doc/setup-binderhub.rst

```diff
@@ -1,104 +1,147 @@
 Set up BinderHub
 ================
 
-.. note::
-
-   If you wish to use encryption with ``letsencrypt``, BinderHub requires
-   that you configure the DNS of a web address to point to the BinderHub IP
-   address. This means that you need to have control over a URL such as
-   ``www.mydomain.com``.
-
-Download the Helm Chart and deployment repo for BinderHub
----------------------------------------------------------
-
 BinderHub uses Helm Charts to set up the applications we'll use in our Binder
 deployment. If you're curious about what Helm Charts are and how they're
 used here, see the `Zero to JupyterHub guide
 <https://zero-to-jupyterhub.readthedocs.io/en/latest/tools.html#helm>`_.
 
-First grab the latest helm chart for BinderHub and install it.::
-
-    helm repo add jupyterhub https://jupyterhub.github.io/helm-chart
-    helm install --name binder jupyterhub/binderhub --version v0.1.0-464c4f5
-
-
-Configure DNS
--------------
+Below we'll cover how to configure your Helm Chart, and how to create your
+BinderHub deployment.
 
-Next we'll configure our web address to point to the IP address for our
-BinderHub.
+Preparing to install
+--------------------
 
-Find the ``EXTERNAL-IP`` of the **nginx-ingress-controller**::
+To configure the Helm Chart we'll need to generate and insert a few pieces of
+information.
 
-    kubectl --namespace=support get svc support-nginx-ingress-controller
+The first is the content of the JSON file created when we set up
```

Comment:
Subheading: JSON file for the **container registry password**

A JSON file was created when we set up the container registry. For more information... ~~We'll copy...below~~